openedx/features/ucsd_features/additional_registration_fields.py

```diff
@@ -13,12 +13,27 @@ class AdditionalRegistrationFieldsForm(ModelForm):
     """
     The fields on this form are derived from the AdditionalRegistrationFields model in models.py.
     """
-    def __init__(self, *args, **kwargs):
-        super(AdditionalRegistrationFieldsForm, self).__init__(*args, **kwargs)
 
     class Meta(object):
         model = AdditionalRegistrationFields
         fields = ('gender_nb', 'ethnicity', 'age', 'education', 'howheard')
+        error_messages = {
+            'gender_nb': {
+                'required': 'Please select your Gender.',
+            },
+            'ethnicity': {
+                'required': 'Please select your Ethnicity.',
+            },
+            'age': {
+                'required': 'Please select your Age Group.',
+            },
+            'education': {
+                'required': 'Please select your Eduction.',
+            },
+            'howheard': {
+                'required': 'Please tell us How did you hear about us.'
```

Comment:
```suggestion
                'required': 'Please tell, how did you hear about us.'
```
sounds good?

Reply:
I don't think this makes much of a difference.

Reply:
Firstly the 'H' of how needs to be small, secondly jo can use the sentence as you formed if was just trying to ignore two 'us' from the sentence, it's just an opinion

Reply:
updated